spec/consensus/proposer-based-timestamp/README.md

```diff
@@ -32,6 +32,22 @@ This assumption is restricted to `Proposal` messages, broadcast by proposers.
 that define whether the timestamp of a block is acceptable,
 according with the introduced `timely` predicate.
 
+#### Note on Liveness
+
+Setting too much conservative (small) synchronous parameters can compromise,
+possibly in an irreversible way, liveness of consensus.
+This is particularly relevant for the `MSGDELAY` parameter.
+When this end-to-end delay is underestimated or unrealistic, proposed block
+times can be rejected by all correct nodes.
```

Comment:
I am not sure "conservative" is the best term here. I think people can have different intuitions what is conservative in this case. I would just stick to "small" here.